controllers/workspace/status.go

```diff
@@ -38,13 +38,11 @@ type currentStatus struct {
 	phase dw.DevWorkspacePhase
 }
 
-func initCurrentStatus() currentStatus {
-	return currentStatus{
-		workspaceConditions: workspaceConditions{
-			conditions: map[dw.DevWorkspaceConditionType]dw.DevWorkspaceCondition{},
-		},
-		phase: dw.DevWorkspaceStatusStarting,
+func newStatusWithPhase(phase dw.DevWorkspacePhase) currentStatus {
+	status := currentStatus{
+		phase: phase,
 	}
+	return status
 }
```

Comment:
Nit: This function could just be replaced with `myStatus := currentStatus{phase: myPhase}` at call sites. Do we need this wrapper over a struct initialization?

Reply:
initially, it also initialized conditions with empty map. Not it makes sense just inline the struct init.

Reply:
done!